modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2737,6 +2742,7 @@ export class Wallet implements IWallet {
             nonce: params.nonce,
             receiveAddress: params.receiveAddress,
             feeOptions,
+            isRebuild: !!params.isRebuild,
```

Comment:
This should always be false right since fill nonce is trying to fill a specific nonce?